docs/serving/samples/hello-world/helloworld-php/index.md

```diff
@@ -5,4 +5,209 @@ weight: 1
 type: "docs"
 ---
 
-{{% readfile file="README.md" %}}
+A simple web app written in PHP that you can use for testing. It reads in an env
+variable `TARGET` and prints `Hello ${TARGET}!`. If `TARGET` is not specified,
+it will use `World` as the `TARGET`.
+
+Follow the steps below to create the sample code and then deploy the app to your
+cluster. You can also download a working copy of the sample, by running the
+following commands:
+
+```shell
+git clone -b "{{< branch >}}" https://github.com/knative/docs knative-docs
+cd knative-docs/docs/serving/samples/hello-world/helloworld-php
+```
+
+## Before you begin
+
+- A Kubernetes cluster with Knative installed and DNS configured. Follow the
+  [installation instructions](../../../../install/README.md) if you need to
+  create one.
+- [Docker](https://www.docker.com) installed and running on your local machine,
+  and a Docker Hub account configured (we'll use it for a container registry).
+
+## Recreating the sample code
+
+1. Create a new directory and cd into it:
+
+   ```shell
+   mkdir app
+   cd app
+   ```
+
+1. Create a file named `index.php` and copy the code block below into it:
+
+   ```php
+   <?php
+   $target = getenv('TARGET', true) ?: 'World';
+   echo sprintf("Hello %s!", $target);
+   ?>
```

Comment:
```suggestion
   <?php
   $target = getenv('TARGET', true) ?: 'World';
   echo sprintf("Hello %s!\n", $target);
   ?>
```

@maximilien could you please apply this diff ? This would make the unit test happy (and possibly also @abrennan89 :)

Reply:
I was able to add the commit @rhuss but you may need to add consent for the CLA :wink:

Reply:
Oh, looks like it worked OK, waiting for the other tests